[drogon] build error in manifest mode

Package: drogon[core,ctl,orm,redis]:x64-windows -> 1.8.4#1

**Host Environment**

- Host: x64-windows
- Compiler: MSVC 19.36.32535.0
-    vcpkg-tool version: 2023-06-15-8c254a5fb6c503615834fc66bd0717664a339a2e
    vcpkg-scripts version: fba81a6a5 2023-07-14 (2 days ago)

**To Reproduce**

`vcpkg install `

**NOTE**: `vcpkg install drogon[core,ctl,orm,redis]:x64-windows` works just fine, and this issue only happens when using manifest mode and having drogon's `ctl` feature in dependencies.

**Failure logs**

```
-- Using cached an-tao-drogon-v1.8.4.tar.gz.
-- Cleaning sources at C:/vcpkg/buildtrees/drogon/src/v1.8.4-f70ee9212f.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source C:/vcpkg/downloads/an-tao-drogon-v1.8.4.tar.gz
-- Applying patch vcpkg.patch
-- Applying patch drogon_config.patch
-- Applying patch fix_gcc13.patch
-- Using source at C:/vcpkg/buildtrees/drogon/src/v1.8.4-f70ee9212f.clean
-- Configuring x64-windows-dbg
-- Configuring x64-windows-rel
-- Building x64-windows-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:134 (message):
    Command failed: C:/devtools/CMake/bin/cmake.exe --build . --config Debug --target install -- -v -j21
    Working Directory: C:/vcpkg/buildtrees/drogon/x64-windows-dbg
    See logs for more information:
      C:\vcpkg\buildtrees\drogon\install-x64-windows-dbg-out.log

Call Stack (most recent call first):
  C:/temp/sample-project/build/vcpkg_installed/x64-windows/share/vcpkg-cmake/vcpkg_cmake_build.cmake:74 (vcpkg_execute_build_process)
  C:/temp/sample-project/build/vcpkg_installed/x64-windows/share/vcpkg-cmake/vcpkg_cmake_install.cmake:16 (vcpkg_cmake_build)
  ports/drogon/portfile.cmake:40 (vcpkg_cmake_install)
  scripts/ports.cmake:147 (include)



```
<details><summary>C:\vcpkg\buildtrees\drogon\install-x64-windows-dbg-out.log</summary>

```
[1/129] C:\PROGRA~1\MICROS~2\2022\COMMUN~1\VC\Tools\MSVC\1436~1.325\bin\Hostx64\x64\cl.exe   /TP -DUSE_BROTLI -DUSE_REDIS -Ddrogon_EXPORTS -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\lib\inc -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\orm_lib\inc -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\nosql_lib\redis\inc -IC:\vcpkg\buildtrees\drogon\x64-windows-dbg -IC:\vcpkg\buildtrees\drogon\x64-windows-dbg\exports -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\third_party\mman-win32 -external:IC:\temp\sample-project\build\vcpkg_installed\x64-windows\include -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /utf-8 /wd4251 /wd4275 -std:c++20 /showIncludes /FoCMakeFiles\drogon.dir\lib\src\CacheFile.cc.obj /FdCMakeFiles\drogon.dir\ /FS -c C:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\lib\src\CacheFile.cc
[2/129] C:\PROGRA~1\MICROS~2\2022\COMMUN~1\VC\Tools\MSVC\1436~1.325\bin\Hostx64\x64\cl.exe   /TP -DUSE_BROTLI -DUSE_REDIS -Ddrogon_EXPORTS -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\lib\inc -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\orm_lib\inc -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\nosql_lib\redis\inc -IC:\vcpkg\buildtrees\drogon\x64-windows-dbg -IC:\vcpkg\buildtrees\drogon\x64-windows-dbg\exports -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\third_party\mman-win32 -external:IC:\temp\sample-project\build\vcpkg_installed\x64-windows\include -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /utf-8 /wd4251 /wd4275 -std:c++20 /showIncludes /FoCMakeFiles\drogon.dir\lib\src\Cookie.cc.obj /FdCMakeFiles\drogon.dir\ /FS -c C:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\lib\src\Cookie.cc
[3/129] C:\PROGRA~1\MICROS~2\2022\COMMUN~1\VC\Tools\MSVC\1436~1.325\bin\Hostx64\x64\cl.exe   /TP -DUSE_BROTLI -DUSE_REDIS -Ddrogon_EXPORTS -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\lib\inc -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\orm_lib\inc -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\nosql_lib\redis\inc -IC:\vcpkg\buildtrees\drogon\x64-windows-dbg -IC:\vcpkg\buildtrees\drogon\x64-windows-dbg\exports -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\third_party\mman-win32 -external:IC:\temp\sample-project\build\vcpkg_installed\x64-windows\include -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /utf-8 /wd4251 /wd4275 -std:c++20 /showIncludes /FoCMakeFiles\drogon.dir\lib\src\HttpViewData.cc.obj /FdCMakeFiles\drogon.dir\ /FS -c C:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\lib\src\HttpViewData.cc
[4/129] C:\PROGRA~1\MICROS~2\2022\COMMUN~1\VC\Tools\MSVC\1436~1.325\bin\Hostx64\x64\cl.exe   /TP -DUSE_BROTLI -DUSE_REDIS -Ddrogon_EXPORTS -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\lib\inc -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\orm_lib\inc -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\nosql_lib\redis\inc -IC:\vcpkg\buildtrees\drogon\x64-windows-dbg -IC:\vcpkg\buildtrees\drogon\x64-windows-dbg\exports -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\third_party\mman-win32 -external:IC:\temp\sample-project\build\vcpkg_installed\x64-windows\include -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /utf-8 /wd4251 /wd4275 -std:c++20 /showIncludes /FoCMakeFiles\drogon.dir\lib\src\DrClassMap.cc.obj /FdCMakeFiles\drogon.dir\ /FS -c C:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\lib\src\DrClassMap.cc
...
Skipped 57 lines
...

[52/129] C:\PROGRA~1\MICROS~2\2022\COMMUN~1\VC\Tools\MSVC\1436~1.325\bin\Hostx64\x64\cl.exe   /TP -DUSE_BROTLI -DUSE_REDIS -Ddrogon_EXPORTS -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\lib\inc -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\orm_lib\inc -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\nosql_lib\redis\inc -IC:\vcpkg\buildtrees\drogon\x64-windows-dbg -IC:\vcpkg\buildtrees\drogon\x64-windows-dbg\exports -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\third_party\mman-win32 -external:IC:\temp\sample-project\build\vcpkg_installed\x64-windows\include -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /utf-8 /wd4251 /wd4275 -std:c++20 /showIncludes /FoCMakeFiles\drogon.dir\nosql_lib\redis\src\RedisClientManager.cc.obj /FdCMakeFiles\drogon.dir\ /FS -c C:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\nosql_lib\redis\src\RedisClientManager.cc
[53/129] C:\PROGRA~1\MICROS~2\2022\COMMUN~1\VC\Tools\MSVC\1436~1.325\bin\Hostx64\x64\cl.exe   /TP -DUSE_BROTLI -DUSE_REDIS -Ddrogon_EXPORTS -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\lib\inc -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\orm_lib\inc -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\nosql_lib\redis\inc -IC:\vcpkg\buildtrees\drogon\x64-windows-dbg -IC:\vcpkg\buildtrees\drogon\x64-windows-dbg\exports -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\third_party\mman-win32 -external:IC:\temp\sample-project\build\vcpkg_installed\x64-windows\include -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /utf-8 /wd4251 /wd4275 -std:c++20 /showIncludes /FoCMakeFiles\drogon.dir\orm_lib\src\Result.cc.obj /FdCMakeFiles\drogon.dir\ /FS -c C:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\orm_lib\src\Result.cc
[54/129] C:\PROGRA~1\MICROS~2\2022\COMMUN~1\VC\Tools\MSVC\1436~1.325\bin\Hostx64\x64\cl.exe   /TP -DUSE_BROTLI -DUSE_REDIS -Ddrogon_EXPORTS -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\lib\inc -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\orm_lib\inc -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\nosql_lib\redis\inc -IC:\vcpkg\buildtrees\drogon\x64-windows-dbg -IC:\vcpkg\buildtrees\drogon\x64-windows-dbg\exports -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\third_party\mman-win32 -external:IC:\temp\sample-project\build\vcpkg_installed\x64-windows\include -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /utf-8 /wd4251 /wd4275 -std:c++20 /showIncludes /FoCMakeFiles\drogon.dir\lib\src\Hodor.cc.obj /FdCMakeFiles\drogon.dir\ /FS -c C:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\lib\src\Hodor.cc
[55/129] C:\PROGRA~1\MICROS~2\2022\COMMUN~1\VC\Tools\MSVC\1436~1.325\bin\Hostx64\x64\cl.exe   /TP -DUSE_BROTLI -DUSE_REDIS -Ddrogon_EXPORTS -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\lib\inc -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\orm_lib\inc -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\nosql_lib\redis\inc -IC:\vcpkg\buildtrees\drogon\x64-windows-dbg -IC:\vcpkg\buildtrees\drogon\x64-windows-dbg\exports -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\third_party\mman-win32 -external:IC:\temp\sample-project\build\vcpkg_installed\x64-windows\include -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /utf-8 /wd4251 /wd4275 -std:c++20 /showIncludes /FoCMakeFiles\drogon.dir\nosql_lib\redis\src\RedisResult.cc.obj /FdCMakeFiles\drogon.dir\ /FS -c C:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\nosql_lib\redis\src\RedisResult.cc
[56/129] C:\PROGRA~1\MICROS~2\2022\COMMUN~1\VC\Tools\MSVC\1436~1.325\bin\Hostx64\x64\cl.exe   /TP -DUSE_BROTLI -DUSE_REDIS -Ddrogon_EXPORTS -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\lib\inc -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\orm_lib\inc -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\nosql_lib\redis\inc -IC:\vcpkg\buildtrees\drogon\x64-windows-dbg -IC:\vcpkg\buildtrees\drogon\x64-windows-dbg\exports -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\third_party\mman-win32 -external:IC:\temp\sample-project\build\vcpkg_installed\x64-windows\include -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /utf-8 /wd4251 /wd4275 -std:c++20 /showIncludes /FoCMakeFiles\drogon.dir\orm_lib\src\Field.cc.obj /FdCMakeFiles\drogon.dir\ /FS -c C:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\orm_lib\src\Field.cc
[57/129] C:\PROGRA~1\MICROS~2\2022\COMMUN~1\VC\Tools\MSVC\1436~1.325\bin\Hostx64\x64\cl.exe   /TP -DUSE_BROTLI -DUSE_REDIS -Ddrogon_EXPORTS -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\lib\inc -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\orm_lib\inc -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\nosql_lib\redis\inc -IC:\vcpkg\buildtrees\drogon\x64-windows-dbg -IC:\vcpkg\buildtrees\drogon\x64-windows-dbg\exports -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\third_party\mman-win32 -external:IC:\temp\sample-project\build\vcpkg_installed\x64-windows\include -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /utf-8 /wd4251 /wd4275 -std:c++20 /showIncludes /FoCMakeFiles\drogon.dir\orm_lib\src\DbClient.cc.obj /FdCMakeFiles\drogon.dir\ /FS -c C:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\orm_lib\src\DbClient.cc
[58/129] C:\PROGRA~1\MICROS~2\2022\COMMUN~1\VC\Tools\MSVC\1436~1.325\bin\Hostx64\x64\cl.exe   /TP -DUSE_BROTLI -DUSE_REDIS -Ddrogon_EXPORTS -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\lib\inc -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\orm_lib\inc -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\nosql_lib\redis\inc -IC:\vcpkg\buildtrees\drogon\x64-windows-dbg -IC:\vcpkg\buildtrees\drogon\x64-windows-dbg\exports -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\third_party\mman-win32 -external:IC:\temp\sample-project\build\vcpkg_installed\x64-windows\include -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /utf-8 /wd4251 /wd4275 -std:c++20 /showIncludes /FoCMakeFiles\drogon.dir\nosql_lib\redis\src\RedisClientImpl.cc.obj /FdCMakeFiles\drogon.dir\ /FS -c C:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\nosql_lib\redis\src\RedisClientImpl.cc
[59/129] C:\PROGRA~1\MICROS~2\2022\COMMUN~1\VC\Tools\MSVC\1436~1.325\bin\Hostx64\x64\cl.exe   /TP -DUSE_BROTLI -DUSE_REDIS -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\lib\inc -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\orm_lib\inc -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\nosql_lib\redis\inc -IC:\vcpkg\buildtrees\drogon\x64-windows-dbg -IC:\vcpkg\buildtrees\drogon\x64-windows-dbg\exports -external:IC:\temp\sample-project\build\vcpkg_installed\x64-windows\include -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /utf-8 /wd4251 /wd4275 -std:c++20 /showIncludes /Fodrogon_ctl\CMakeFiles\_drogon_ctl.dir\main.cc.obj /Fddrogon_ctl\CMakeFiles\_drogon_ctl.dir\ /FS -c C:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\drogon_ctl\main.cc
[60/129] C:\PROGRA~1\MICROS~2\2022\COMMUN~1\VC\Tools\MSVC\1436~1.325\bin\Hostx64\x64\cl.exe   /TP -DUSE_BROTLI -DUSE_REDIS -Ddrogon_EXPORTS -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\lib\inc -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\orm_lib\inc -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\nosql_lib\redis\inc -IC:\vcpkg\buildtrees\drogon\x64-windows-dbg -IC:\vcpkg\buildtrees\drogon\x64-windows-dbg\exports -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\third_party\mman-win32 -external:IC:\temp\sample-project\build\vcpkg_installed\x64-windows\include -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /utf-8 /wd4251 /wd4275 -std:c++20 /showIncludes /FoCMakeFiles\drogon.dir\orm_lib\src\Criteria.cc.obj /FdCMakeFiles\drogon.dir\ /FS -c C:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\orm_lib\src\Criteria.cc
[61/129] C:\PROGRA~1\MICROS~2\2022\COMMUN~1\VC\Tools\MSVC\1436~1.325\bin\Hostx64\x64\cl.exe   /TP -DUSE_BROTLI -DUSE_REDIS -Ddrogon_EXPORTS -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\lib\inc -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\orm_lib\inc -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\nosql_lib\redis\inc -IC:\vcpkg\buildtrees\drogon\x64-windows-dbg -IC:\vcpkg\buildtrees\drogon\x64-windows-dbg\exports -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\third_party\mman-win32 -external:IC:\temp\sample-project\build\vcpkg_installed\x64-windows\include -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /utf-8 /wd4251 /wd4275 -std:c++20 /showIncludes /FoCMakeFiles\drogon.dir\nosql_lib\redis\src\RedisTransactionImpl.cc.obj /FdCMakeFiles\drogon.dir\ /FS -c C:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\nosql_lib\redis\src\RedisTransactionImpl.cc
[62/129] C:\PROGRA~1\MICROS~2\2022\COMMUN~1\VC\Tools\MSVC\1436~1.325\bin\Hostx64\x64\cl.exe   /TP -DUSE_BROTLI -DUSE_REDIS -Ddrogon_EXPORTS -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\lib\inc -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\orm_lib\inc -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\nosql_lib\redis\inc -IC:\vcpkg\buildtrees\drogon\x64-windows-dbg -IC:\vcpkg\buildtrees\drogon\x64-windows-dbg\exports -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\third_party\mman-win32 -external:IC:\temp\sample-project\build\vcpkg_installed\x64-windows\include -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /utf-8 /wd4251 /wd4275 -std:c++20 /showIncludes /FoCMakeFiles\drogon.dir\orm_lib\src\DbConnection.cc.obj /FdCMakeFiles\drogon.dir\ /FS -c C:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\orm_lib\src\DbConnection.cc
[63/129] C:\PROGRA~1\MICROS~2\2022\COMMUN~1\VC\Tools\MSVC\1436~1.325\bin\Hostx64\x64\cl.exe   /TP -DUSE_BROTLI -DUSE_REDIS -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\lib\inc -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\orm_lib\inc -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\nosql_lib\redis\inc -IC:\vcpkg\buildtrees\drogon\x64-windows-dbg -IC:\vcpkg\buildtrees\drogon\x64-windows-dbg\exports -external:IC:\temp\sample-project\build\vcpkg_installed\x64-windows\include -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /utf-8 /wd4251 /wd4275 -std:c++20 /showIncludes /Fodrogon_ctl\CMakeFiles\_drogon_ctl.dir\cmd.cc.obj /Fddrogon_ctl\CMakeFiles\_drogon_ctl.dir\ /FS -c C:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\drogon_ctl\cmd.cc
[64/129] C:\PROGRA~1\MICROS~2\2022\COMMUN~1\VC\Tools\MSVC\1436~1.325\bin\Hostx64\x64\cl.exe   /TP -DUSE_BROTLI -DUSE_REDIS -Ddrogon_EXPORTS -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\lib\inc -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\orm_lib\inc -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\nosql_lib\redis\inc -IC:\vcpkg\buildtrees\drogon\x64-windows-dbg -IC:\vcpkg\buildtrees\drogon\x64-windows-dbg\exports -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\third_party\mman-win32 -external:IC:\temp\sample-project\build\vcpkg_installed\x64-windows\include -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /utf-8 /wd4251 /wd4275 -std:c++20 /showIncludes /FoCMakeFiles\drogon.dir\nosql_lib\redis\src\RedisConnection.cc.obj /FdCMakeFiles\drogon.dir\ /FS -c C:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\nosql_lib\redis\src\RedisConnection.cc
C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.36.32532\include\memory(3465): warning C4244: 'argument': conversion from 'redisFD' to 'int', possible loss of data
C:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\nosql_lib\redis\src\RedisConnection.cc(72): note: see reference to function template instantiation 'std::unique_ptr<trantor::Channel,std::default_delete<trantor::Channel>> std::make_unique<trantor::Channel,trantor::EventLoop*&,redisFD&,0>(trantor::EventLoop *&,redisFD &)' being compiled
[65/129] C:\PROGRA~1\MICROS~2\2022\COMMUN~1\VC\Tools\MSVC\1436~1.325\bin\Hostx64\x64\cl.exe   /TP -DUSE_BROTLI -DUSE_REDIS -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\lib\inc -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\orm_lib\inc -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\nosql_lib\redis\inc -IC:\vcpkg\buildtrees\drogon\x64-windows-dbg -IC:\vcpkg\buildtrees\drogon\x64-windows-dbg\exports -external:IC:\temp\sample-project\build\vcpkg_installed\x64-windows\include -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /utf-8 /wd4251 /wd4275 -std:c++20 /showIncludes /Fodrogon_ctl\CMakeFiles\_drogon_ctl.dir\create.cc.obj /Fddrogon_ctl\CMakeFiles\_drogon_ctl.dir\ /FS -c C:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\drogon_ctl\create.cc
[66/129] C:\PROGRA~1\MICROS~2\2022\COMMUN~1\VC\Tools\MSVC\1436~1.325\bin\Hostx64\x64\cl.exe   /TP -DUSE_BROTLI -DUSE_REDIS -Ddrogon_EXPORTS -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\lib\inc -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\orm_lib\inc -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\nosql_lib\redis\inc -IC:\vcpkg\buildtrees\drogon\x64-windows-dbg -IC:\vcpkg\buildtrees\drogon\x64-windows-dbg\exports -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\third_party\mman-win32 -external:IC:\temp\sample-project\build\vcpkg_installed\x64-windows\include -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /utf-8 /wd4251 /wd4275 -std:c++20 /showIncludes /FoCMakeFiles\drogon.dir\orm_lib\src\Row.cc.obj /FdCMakeFiles\drogon.dir\ /FS -c C:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\orm_lib\src\Row.cc
[67/129] C:\PROGRA~1\MICROS~2\2022\COMMUN~1\VC\Tools\MSVC\1436~1.325\bin\Hostx64\x64\cl.exe   /TP -DUSE_BROTLI -DUSE_REDIS -Ddrogon_EXPORTS -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\lib\inc -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\orm_lib\inc -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\nosql_lib\redis\inc -IC:\vcpkg\buildtrees\drogon\x64-windows-dbg -IC:\vcpkg\buildtrees\drogon\x64-windows-dbg\exports -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\third_party\mman-win32 -external:IC:\temp\sample-project\build\vcpkg_installed\x64-windows\include -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /utf-8 /wd4251 /wd4275 -std:c++20 /showIncludes /FoCMakeFiles\drogon.dir\nosql_lib\redis\src\RedisSubscriberImpl.cc.obj /FdCMakeFiles\drogon.dir\ /FS -c C:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\nosql_lib\redis\src\RedisSubscriberImpl.cc
[68/129] C:\PROGRA~1\MICROS~2\2022\COMMUN~1\VC\Tools\MSVC\1436~1.325\bin\Hostx64\x64\cl.exe   /TP -DUSE_BROTLI -DUSE_REDIS -Ddrogon_EXPORTS -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\lib\inc -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\orm_lib\inc -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\nosql_lib\redis\inc -IC:\vcpkg\buildtrees\drogon\x64-windows-dbg -IC:\vcpkg\buildtrees\drogon\x64-windows-dbg\exports -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\third_party\mman-win32 -external:IC:\temp\sample-project\build\vcpkg_installed\x64-windows\include -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /utf-8 /wd4251 /wd4275 -std:c++20 /showIncludes /FoCMakeFiles\drogon.dir\nosql_lib\redis\src\SubscribeContext.cc.obj /FdCMakeFiles\drogon.dir\ /FS -c C:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\nosql_lib\redis\src\SubscribeContext.cc
[69/129] C:\PROGRA~1\MICROS~2\2022\COMMUN~1\VC\Tools\MSVC\1436~1.325\bin\Hostx64\x64\cl.exe   /TP -DUSE_BROTLI -DUSE_REDIS -Ddrogon_EXPORTS -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\lib\inc -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\orm_lib\inc -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\nosql_lib\redis\inc -IC:\vcpkg\buildtrees\drogon\x64-windows-dbg -IC:\vcpkg\buildtrees\drogon\x64-windows-dbg\exports -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\third_party\mman-win32 -external:IC:\temp\sample-project\build\vcpkg_installed\x64-windows\include -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /utf-8 /wd4251 /wd4275 -std:c++20 /showIncludes /FoCMakeFiles\drogon.dir\orm_lib\src\DbClientLockFree.cc.obj /FdCMakeFiles\drogon.dir\ /FS -c C:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\orm_lib\src\DbClientLockFree.cc
[70/129] C:\PROGRA~1\MICROS~2\2022\COMMUN~1\VC\Tools\MSVC\1436~1.325\bin\Hostx64\x64\cl.exe   /TP -DUSE_BROTLI -DUSE_REDIS -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\lib\inc -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\orm_lib\inc -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\nosql_lib\redis\inc -IC:\vcpkg\buildtrees\drogon\x64-windows-dbg -IC:\vcpkg\buildtrees\drogon\x64-windows-dbg\exports -external:IC:\temp\sample-project\build\vcpkg_installed\x64-windows\include -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /utf-8 /wd4251 /wd4275 -std:c++20 /showIncludes /Fodrogon_ctl\CMakeFiles\_drogon_ctl.dir\create_view.cc.obj /Fddrogon_ctl\CMakeFiles\_drogon_ctl.dir\ /FS -c C:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\drogon_ctl\create_view.cc
[71/129] C:\PROGRA~1\MICROS~2\2022\COMMUN~1\VC\Tools\MSVC\1436~1.325\bin\Hostx64\x64\cl.exe   /TP -DUSE_BROTLI -DUSE_REDIS -Ddrogon_EXPORTS -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\lib\inc -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\orm_lib\inc -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\nosql_lib\redis\inc -IC:\vcpkg\buildtrees\drogon\x64-windows-dbg -IC:\vcpkg\buildtrees\drogon\x64-windows-dbg\exports -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\third_party\mman-win32 -external:IC:\temp\sample-project\build\vcpkg_installed\x64-windows\include -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /utf-8 /wd4251 /wd4275 -std:c++20 /showIncludes /FoCMakeFiles\drogon.dir\orm_lib\src\DbClientImpl.cc.obj /FdCMakeFiles\drogon.dir\ /FS -c C:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\orm_lib\src\DbClientImpl.cc
[72/129] C:\PROGRA~1\MICROS~2\2022\COMMUN~1\VC\Tools\MSVC\1436~1.325\bin\Hostx64\x64\cl.exe   /TP -DUSE_BROTLI -DUSE_REDIS -Ddrogon_EXPORTS -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\lib\inc -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\orm_lib\inc -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\nosql_lib\redis\inc -IC:\vcpkg\buildtrees\drogon\x64-windows-dbg -IC:\vcpkg\buildtrees\drogon\x64-windows-dbg\exports -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\third_party\mman-win32 -external:IC:\temp\sample-project\build\vcpkg_installed\x64-windows\include -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /utf-8 /wd4251 /wd4275 -std:c++20 /showIncludes /FoCMakeFiles\drogon.dir\orm_lib\src\RestfulController.cc.obj /FdCMakeFiles\drogon.dir\ /FS -c C:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\orm_lib\src\RestfulController.cc
[73/129] C:\PROGRA~1\MICROS~2\2022\COMMUN~1\VC\Tools\MSVC\1436~1.325\bin\Hostx64\x64\cl.exe   /TP -DUSE_BROTLI -DUSE_REDIS -Ddrogon_EXPORTS -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\lib\inc -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\orm_lib\inc -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\nosql_lib\redis\inc -IC:\vcpkg\buildtrees\drogon\x64-windows-dbg -IC:\vcpkg\buildtrees\drogon\x64-windows-dbg\exports -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\third_party\mman-win32 -external:IC:\temp\sample-project\build\vcpkg_installed\x64-windows\include -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /utf-8 /wd4251 /wd4275 -std:c++20 /showIncludes /FoCMakeFiles\drogon.dir\orm_lib\src\TransactionImpl.cc.obj /FdCMakeFiles\drogon.dir\ /FS -c C:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\orm_lib\src\TransactionImpl.cc
[74/129] C:\PROGRA~1\MICROS~2\2022\COMMUN~1\VC\Tools\MSVC\1436~1.325\bin\Hostx64\x64\cl.exe   /TP -DUSE_BROTLI -DUSE_REDIS -Ddrogon_EXPORTS -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\lib\inc -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\orm_lib\inc -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\nosql_lib\redis\inc -IC:\vcpkg\buildtrees\drogon\x64-windows-dbg -IC:\vcpkg\buildtrees\drogon\x64-windows-dbg\exports -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\third_party\mman-win32 -external:IC:\temp\sample-project\build\vcpkg_installed\x64-windows\include -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /utf-8 /wd4251 /wd4275 -std:c++20 /showIncludes /FoCMakeFiles\drogon.dir\orm_lib\src\SqlBinder.cc.obj /FdCMakeFiles\drogon.dir\ /FS -c C:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\orm_lib\src\SqlBinder.cc
[75/129] C:\PROGRA~1\MICROS~2\2022\COMMUN~1\VC\Tools\MSVC\1436~1.325\bin\Hostx64\x64\cl.exe   /TP -DUSE_BROTLI -DUSE_REDIS -Ddrogon_EXPORTS -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\lib\inc -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\orm_lib\inc -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\nosql_lib\redis\inc -IC:\vcpkg\buildtrees\drogon\x64-windows-dbg -IC:\vcpkg\buildtrees\drogon\x64-windows-dbg\exports -IC:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\third_party\mman-win32 -external:IC:\temp\sample-project\build\vcpkg_installed\x64-windows\include -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /utf-8 /wd4251 /wd4275 -std:c++20 /showIncludes /FoCMakeFiles\drogon.dir\lib\src\DbClientManagerSkipped.cc.obj /FdCMakeFiles\drogon.dir\ /FS -c C:\vcpkg\buildtrees\drogon\src\v1.8.4-f70ee9212f.clean\lib\src\DbClientManagerSkipped.cc
[76/129] cmd.exe /C "cd . && C:\devtools\CMake\bin\cmake.exe -E vs_link_dll --intdir=CMakeFiles\drogon.dir --rc=C:\PROGRA~2\WI3CF2~1\10\bin\100226~1.0\x64\rc.exe --mt=C:\PROGRA~2\WI3CF2~1\10\bin\100226~1.0\x64\mt.exe --manifests  -- C:\PROGRA~1\MICROS~2\2022\COMMUN~1\VC\Tools\MSVC\1436~1.325\bin\Hostx64\x64\link.exe  CMakeFiles\drogon.dir\lib\src\AOPAdvice.cc.obj CMakeFiles\drogon.dir\lib\src\CacheFile.cc.obj CMakeFiles\drogon.dir\lib\src\ConfigLoader.cc.obj CMakeFiles\drogon.dir\lib\src\Cookie.cc.obj CMakeFiles\drogon.dir\lib\src\DrClassMap.cc.obj CMakeFiles\drogon.dir\lib\src\DrTemplateBase.cc.obj CMakeFiles\drogon.dir\lib\src\FiltersFunction.cc.obj CMakeFiles\drogon.dir\lib\src\HttpAppFrameworkImpl.cc.obj CMakeFiles\drogon.dir\lib\src\HttpBinder.cc.obj CMakeFiles\drogon.dir\lib\src\HttpClientImpl.cc.obj CMakeFiles\drogon.dir\lib\src\HttpControllersRouter.cc.obj CMakeFiles\drogon.dir\lib\src\HttpFileImpl.cc.obj CMakeFiles\drogon.dir\lib\src\HttpFileUploadRequest.cc.obj CMakeFiles\drogon.dir\lib\src\HttpRequestImpl.cc.obj CMakeFiles\drogon.dir\lib\src\HttpRequestParser.cc.obj CMakeFiles\drogon.dir\lib\src\HttpResponseImpl.cc.obj CMakeFiles\drogon.dir\lib\src\HttpResponseParser.cc.obj CMakeFiles\drogon.dir\lib\src\HttpServer.cc.obj CMakeFiles\drogon.dir\lib\src\HttpSimpleControllersRouter.cc.obj CMakeFiles\drogon.dir\lib\src\HttpUtils.cc.obj CMakeFiles\drogon.dir\lib\src\HttpViewData.cc.obj CMakeFiles\drogon.dir\lib\src\IntranetIpFilter.cc.obj CMakeFiles\drogon.dir\lib\src\ListenerManager.cc.obj CMakeFiles\drogon.dir\lib\src\LocalHostFilter.cc.obj CMakeFiles\drogon.dir\lib\src\MultiPart.cc.obj CMakeFiles\drogon.dir\lib\src\NotFound.cc.obj CMakeFiles\drogon.dir\lib\src\PluginsManager.cc.obj CMakeFiles\drogon.dir\lib\src\RangeParser.cc.obj CMakeFiles\drogon.dir\lib\src\SecureSSLRedirector.cc.obj CMakeFiles\drogon.dir\lib\src\AccessLogger.cc.obj CMakeFiles\drogon.dir\lib\src\RealIpResolver.cc.obj CMakeFiles\drogon.dir\lib\src\SessionManager.cc.obj CMakeFiles\drogon.dir\lib\src\StaticFileRouter.cc.obj CMakeFiles\drogon.dir\lib\src\TaskTimeoutFlag.cc.obj CMakeFiles\drogon.dir\lib\src\Utilities.cc.obj CMakeFiles\drogon.dir\lib\src\WebSocketClientImpl.cc.obj CMakeFiles\drogon.dir\lib\src\WebSocketConnectionImpl.cc.obj CMakeFiles\drogon.dir\lib\src\WebsocketControllersRouter.cc.obj CMakeFiles\drogon.dir\lib\src\RateLimiter.cc.obj CMakeFiles\drogon.dir\lib\src\FixedWindowRateLimiter.cc.obj CMakeFiles\drogon.dir\lib\src\SlidingWindowRateLimiter.cc.obj CMakeFiles\drogon.dir\lib\src\TokenBucketRateLimiter.cc.obj CMakeFiles\drogon.dir\lib\src\Hodor.cc.obj CMakeFiles\drogon.dir\lib\src\drogon_test.cc.obj CMakeFiles\drogon.dir\lib\src\ConfigAdapterManager.cc.obj CMakeFiles\drogon.dir\lib\src\JsonConfigAdapter.cc.obj CMakeFiles\drogon.dir\lib\src\YamlConfigAdapter.cc.obj CMakeFiles\drogon.dir\third_party\mman-win32\mman.c.obj CMakeFiles\drogon.dir\nosql_lib\redis\src\RedisClientImpl.cc.obj CMakeFiles\drogon.dir\nosql_lib\redis\src\RedisClientLockFree.cc.obj CMakeFiles\drogon.dir\nosql_lib\redis\src\RedisClientManager.cc.obj CMakeFiles\drogon.dir\nosql_lib\redis\src\RedisConnection.cc.obj CMakeFiles\drogon.dir\nosql_lib\redis\src\RedisResult.cc.obj CMakeFiles\drogon.dir\nosql_lib\redis\src\RedisTransactionImpl.cc.obj CMakeFiles\drogon.dir\nosql_lib\redis\src\SubscribeContext.cc.obj CMakeFiles\drogon.dir\nosql_lib\redis\src\RedisSubscriberImpl.cc.obj CMakeFiles\drogon.dir\orm_lib\src\ArrayParser.cc.obj CMakeFiles\drogon.dir\orm_lib\src\Criteria.cc.obj CMakeFiles\drogon.dir\orm_lib\src\DbClient.cc.obj CMakeFiles\drogon.dir\orm_lib\src\DbClientImpl.cc.obj CMakeFiles\drogon.dir\orm_lib\src\DbClientLockFree.cc.obj CMakeFiles\drogon.dir\orm_lib\src\DbConnection.cc.obj CMakeFiles\drogon.dir\orm_lib\src\DbListener.cc.obj CMakeFiles\drogon.dir\orm_lib\src\Exception.cc.obj CMakeFiles\drogon.dir\orm_lib\src\Field.cc.obj CMakeFiles\drogon.dir\orm_lib\src\Result.cc.obj CMakeFiles\drogon.dir\orm_lib\src\Row.cc.obj CMakeFiles\drogon.dir\orm_lib\src\SqlBinder.cc.obj CMakeFiles\drogon.dir\orm_lib\src\TransactionImpl.cc.obj CMakeFiles\drogon.dir\orm_lib\src\RestfulController.cc.obj CMakeFiles\drogon.dir\lib\src\DbClientManagerSkipped.cc.obj  /out:drogon.dll /implib:drogon.lib /pdb:drogon.pdb /dll /version:1.8 /machine:x64 /nologo    /debug /INCREMENTAL  Rpcrt4.lib  ws2_32.lib  crypt32.lib  Advapi32.lib  C:\temp\sample-project\build\vcpkg_installed\x64-windows\debug\lib\trantor.lib  shlwapi.lib  C:\temp\sample-project\build\vcpkg_installed\x64-windows\debug\lib\zlibd.lib  C:\temp\sample-project\build\vcpkg_installed\x64-windows\debug\lib\libssl.lib  C:\temp\sample-project\build\vcpkg_installed\x64-windows\debug\lib\libcrypto.lib  C:\temp\sample-project\build\vcpkg_installed\x64-windows\debug\lib\jsoncpp.lib  C:\temp\sample-project\build\vcpkg_installed\x64-windows\debug\lib\brotlidec.lib  C:\temp\sample-project\build\vcpkg_installed\x64-windows\debug\lib\brotlienc.lib  C:\temp\sample-project\build\vcpkg_installed\x64-windows\debug\lib\brotlicommon.lib  C:\temp\sample-project\build\vcpkg_installed\x64-windows\lib\hiredis.lib  kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib  && cd ."
[77/129] cmd.exe /C "cd . && C:\devtools\CMake\bin\cmake.exe -E vs_link_exe --intdir=drogon_ctl\CMakeFiles\_drogon_ctl.dir --rc=C:\PROGRA~2\WI3CF2~1\10\bin\100226~1.0\x64\rc.exe --mt=C:\PROGRA~2\WI3CF2~1\10\bin\100226~1.0\x64\mt.exe --manifests  -- C:\PROGRA~1\MICROS~2\2022\COMMUN~1\VC\Tools\MSVC\1436~1.325\bin\Hostx64\x64\link.exe  drogon_ctl\CMakeFiles\_drogon_ctl.dir\main.cc.obj drogon_ctl\CMakeFiles\_drogon_ctl.dir\cmd.cc.obj drogon_ctl\CMakeFiles\_drogon_ctl.dir\create.cc.obj drogon_ctl\CMakeFiles\_drogon_ctl.dir\create_view.cc.obj  /out:drogon_ctl\_drogon_ctl.exe /implib:drogon_ctl\_drogon_ctl.lib /pdb:drogon_ctl\_drogon_ctl.pdb /version:0.0 /machine:x64 /nologo    /debug /INCREMENTAL /subsystem:console  drogon.lib  Rpcrt4.lib  ws2_32.lib  crypt32.lib  Advapi32.lib  C:\temp\sample-project\build\vcpkg_installed\x64-windows\debug\lib\trantor.lib  C:\temp\sample-project\build\vcpkg_installed\x64-windows\debug\lib\jsoncpp.lib  kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib && cmd.exe /C "cd /D C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl && C:\devtools\CMake\bin\cmake.exe -DCTL_FILE=C:/vcpkg/buildtrees/drogon/x64-windows-dbg/drogon.dll -DINSTALL_BIN_DIR=C:/vcpkg/buildtrees/drogon/x64-windows-dbg/drogon_ctl -P C:/vcpkg/buildtrees/drogon/src/v1.8.4-f70ee9212f.clean/drogon_ctl/CopyDlls.cmake && cd /D C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl && C:\devtools\CMake\bin\cmake.exe -DCTL_FILE=C:/temp/sample-project/build/vcpkg_installed/x64-windows/debug/bin/trantor.dll -DINSTALL_BIN_DIR=C:/vcpkg/buildtrees/drogon/x64-windows-dbg/drogon_ctl -P C:/vcpkg/buildtrees/drogon/src/v1.8.4-f70ee9212f.clean/drogon_ctl/CopyDlls.cmake""
[78/129] cmd.exe /C "cd /D C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl && C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl\_drogon_ctl.exe create view C:/vcpkg/buildtrees/drogon/src/v1.8.4-f70ee9212f.clean/drogon_ctl/templates/model_h.csp"
FAILED: drogon_ctl/model_h.h drogon_ctl/model_h.cc C:/vcpkg/buildtrees/drogon/x64-windows-dbg/drogon_ctl/model_h.h C:/vcpkg/buildtrees/drogon/x64-windows-dbg/drogon_ctl/model_h.cc 
cmd.exe /C "cd /D C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl && C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl\_drogon_ctl.exe create view C:/vcpkg/buildtrees/drogon/src/v1.8.4-f70ee9212f.clean/drogon_ctl/templates/model_h.csp"
[79/129] cmd.exe /C "cd /D C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl && C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl\_drogon_ctl.exe create view C:/vcpkg/buildtrees/drogon/src/v1.8.4-f70ee9212f.clean/drogon_ctl/templates/plugin_cc.csp"
FAILED: drogon_ctl/plugin_cc.h drogon_ctl/plugin_cc.cc C:/vcpkg/buildtrees/drogon/x64-windows-dbg/drogon_ctl/plugin_cc.h C:/vcpkg/buildtrees/drogon/x64-windows-dbg/drogon_ctl/plugin_cc.cc 
cmd.exe /C "cd /D C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl && C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl\_drogon_ctl.exe create view C:/vcpkg/buildtrees/drogon/src/v1.8.4-f70ee9212f.clean/drogon_ctl/templates/plugin_cc.csp"
[80/129] cmd.exe /C "cd /D C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl && C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl\_drogon_ctl.exe create view C:/vcpkg/buildtrees/drogon/src/v1.8.4-f70ee9212f.clean/drogon_ctl/templates/restful_controller_base_cc.csp"
FAILED: drogon_ctl/restful_controller_base_cc.h drogon_ctl/restful_controller_base_cc.cc C:/vcpkg/buildtrees/drogon/x64-windows-dbg/drogon_ctl/restful_controller_base_cc.h C:/vcpkg/buildtrees/drogon/x64-windows-dbg/drogon_ctl/restful_controller_base_cc.cc 
cmd.exe /C "cd /D C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl && C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl\_drogon_ctl.exe create view C:/vcpkg/buildtrees/drogon/src/v1.8.4-f70ee9212f.clean/drogon_ctl/templates/restful_controller_base_cc.csp"
[81/129] cmd.exe /C "cd /D C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl && C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl\_drogon_ctl.exe create view C:/vcpkg/buildtrees/drogon/src/v1.8.4-f70ee9212f.clean/drogon_ctl/templates/filter_cc.csp"
FAILED: drogon_ctl/filter_cc.h drogon_ctl/filter_cc.cc C:/vcpkg/buildtrees/drogon/x64-windows-dbg/drogon_ctl/filter_cc.h C:/vcpkg/buildtrees/drogon/x64-windows-dbg/drogon_ctl/filter_cc.cc 
cmd.exe /C "cd /D C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl && C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl\_drogon_ctl.exe create view C:/vcpkg/buildtrees/drogon/src/v1.8.4-f70ee9212f.clean/drogon_ctl/templates/filter_cc.csp"
[82/129] cmd.exe /C "cd /D C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl && C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl\_drogon_ctl.exe create view C:/vcpkg/buildtrees/drogon/src/v1.8.4-f70ee9212f.clean/drogon_ctl/templates/restful_controller_cc.csp"
FAILED: drogon_ctl/restful_controller_cc.h drogon_ctl/restful_controller_cc.cc C:/vcpkg/buildtrees/drogon/x64-windows-dbg/drogon_ctl/restful_controller_cc.h C:/vcpkg/buildtrees/drogon/x64-windows-dbg/drogon_ctl/restful_controller_cc.cc 
cmd.exe /C "cd /D C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl && C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl\_drogon_ctl.exe create view C:/vcpkg/buildtrees/drogon/src/v1.8.4-f70ee9212f.clean/drogon_ctl/templates/restful_controller_cc.csp"
[83/129] cmd.exe /C "cd /D C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl && C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl\_drogon_ctl.exe create view C:/vcpkg/buildtrees/drogon/src/v1.8.4-f70ee9212f.clean/drogon_ctl/templates/test_main.csp"
FAILED: drogon_ctl/test_main.h drogon_ctl/test_main.cc C:/vcpkg/buildtrees/drogon/x64-windows-dbg/drogon_ctl/test_main.h C:/vcpkg/buildtrees/drogon/x64-windows-dbg/drogon_ctl/test_main.cc 
cmd.exe /C "cd /D C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl && C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl\_drogon_ctl.exe create view C:/vcpkg/buildtrees/drogon/src/v1.8.4-f70ee9212f.clean/drogon_ctl/templates/test_main.csp"
[84/129] cmd.exe /C "cd /D C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl && C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl\_drogon_ctl.exe create view C:/vcpkg/buildtrees/drogon/src/v1.8.4-f70ee9212f.clean/drogon_ctl/templates/demoMain.csp"
FAILED: drogon_ctl/demoMain.h drogon_ctl/demoMain.cc C:/vcpkg/buildtrees/drogon/x64-windows-dbg/drogon_ctl/demoMain.h C:/vcpkg/buildtrees/drogon/x64-windows-dbg/drogon_ctl/demoMain.cc 
cmd.exe /C "cd /D C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl && C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl\_drogon_ctl.exe create view C:/vcpkg/buildtrees/drogon/src/v1.8.4-f70ee9212f.clean/drogon_ctl/templates/demoMain.csp"
[85/129] cmd.exe /C "cd /D C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl && C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl\_drogon_ctl.exe create view C:/vcpkg/buildtrees/drogon/src/v1.8.4-f70ee9212f.clean/drogon_ctl/templates/model_cc.csp"
FAILED: drogon_ctl/model_cc.h drogon_ctl/model_cc.cc C:/vcpkg/buildtrees/drogon/x64-windows-dbg/drogon_ctl/model_cc.h C:/vcpkg/buildtrees/drogon/x64-windows-dbg/drogon_ctl/model_cc.cc 
cmd.exe /C "cd /D C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl && C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl\_drogon_ctl.exe create view C:/vcpkg/buildtrees/drogon/src/v1.8.4-f70ee9212f.clean/drogon_ctl/templates/model_cc.csp"
[86/129] cmd.exe /C "cd /D C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl && C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl\_drogon_ctl.exe create view C:/vcpkg/buildtrees/drogon/src/v1.8.4-f70ee9212f.clean/drogon_ctl/templates/restful_controller_h.csp"
FAILED: drogon_ctl/restful_controller_h.h drogon_ctl/restful_controller_h.cc C:/vcpkg/buildtrees/drogon/x64-windows-dbg/drogon_ctl/restful_controller_h.h C:/vcpkg/buildtrees/drogon/x64-windows-dbg/drogon_ctl/restful_controller_h.cc 
cmd.exe /C "cd /D C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl && C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl\_drogon_ctl.exe create view C:/vcpkg/buildtrees/drogon/src/v1.8.4-f70ee9212f.clean/drogon_ctl/templates/restful_controller_h.csp"
[87/129] cmd.exe /C "cd /D C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl && C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl\_drogon_ctl.exe create view C:/vcpkg/buildtrees/drogon/src/v1.8.4-f70ee9212f.clean/drogon_ctl/templates/gitignore.csp"
FAILED: drogon_ctl/gitignore.h drogon_ctl/gitignore.cc C:/vcpkg/buildtrees/drogon/x64-windows-dbg/drogon_ctl/gitignore.h C:/vcpkg/buildtrees/drogon/x64-windows-dbg/drogon_ctl/gitignore.cc 
cmd.exe /C "cd /D C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl && C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl\_drogon_ctl.exe create view C:/vcpkg/buildtrees/drogon/src/v1.8.4-f70ee9212f.clean/drogon_ctl/templates/gitignore.csp"
[88/129] cmd.exe /C "cd /D C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl && C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl\_drogon_ctl.exe create view C:/vcpkg/buildtrees/drogon/src/v1.8.4-f70ee9212f.clean/drogon_ctl/templates/restful_controller_custom_h.csp"
FAILED: drogon_ctl/restful_controller_custom_h.h drogon_ctl/restful_controller_custom_h.cc C:/vcpkg/buildtrees/drogon/x64-windows-dbg/drogon_ctl/restful_controller_custom_h.h C:/vcpkg/buildtrees/drogon/x64-windows-dbg/drogon_ctl/restful_controller_custom_h.cc 
cmd.exe /C "cd /D C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl && C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl\_drogon_ctl.exe create view C:/vcpkg/buildtrees/drogon/src/v1.8.4-f70ee9212f.clean/drogon_ctl/templates/restful_controller_custom_h.csp"
[89/129] cmd.exe /C "cd /D C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl && C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl\_drogon_ctl.exe create view C:/vcpkg/buildtrees/drogon/src/v1.8.4-f70ee9212f.clean/drogon_ctl/templates/plugin_h.csp"
FAILED: drogon_ctl/plugin_h.h drogon_ctl/plugin_h.cc C:/vcpkg/buildtrees/drogon/x64-windows-dbg/drogon_ctl/plugin_h.h C:/vcpkg/buildtrees/drogon/x64-windows-dbg/drogon_ctl/plugin_h.cc 
cmd.exe /C "cd /D C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl && C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl\_drogon_ctl.exe create view C:/vcpkg/buildtrees/drogon/src/v1.8.4-f70ee9212f.clean/drogon_ctl/templates/plugin_h.csp"
[90/129] cmd.exe /C "cd /D C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl && C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl\_drogon_ctl.exe create view C:/vcpkg/buildtrees/drogon/src/v1.8.4-f70ee9212f.clean/drogon_ctl/templates/config.csp"
FAILED: drogon_ctl/config.h drogon_ctl/config.cc C:/vcpkg/buildtrees/drogon/x64-windows-dbg/drogon_ctl/config.h C:/vcpkg/buildtrees/drogon/x64-windows-dbg/drogon_ctl/config.cc 
cmd.exe /C "cd /D C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl && C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl\_drogon_ctl.exe create view C:/vcpkg/buildtrees/drogon/src/v1.8.4-f70ee9212f.clean/drogon_ctl/templates/config.csp"
[91/129] cmd.exe /C "cd /D C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl && C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl\_drogon_ctl.exe create view C:/vcpkg/buildtrees/drogon/src/v1.8.4-f70ee9212f.clean/drogon_ctl/templates/model_json.csp"
FAILED: drogon_ctl/model_json.h drogon_ctl/model_json.cc C:/vcpkg/buildtrees/drogon/x64-windows-dbg/drogon_ctl/model_json.h C:/vcpkg/buildtrees/drogon/x64-windows-dbg/drogon_ctl/model_json.cc 
cmd.exe /C "cd /D C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl && C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl\_drogon_ctl.exe create view C:/vcpkg/buildtrees/drogon/src/v1.8.4-f70ee9212f.clean/drogon_ctl/templates/model_json.csp"
[92/129] cmd.exe /C "cd /D C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl && C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl\_drogon_ctl.exe create view C:/vcpkg/buildtrees/drogon/src/v1.8.4-f70ee9212f.clean/drogon_ctl/templates/test_cmake.csp"
FAILED: drogon_ctl/test_cmake.h drogon_ctl/test_cmake.cc C:/vcpkg/buildtrees/drogon/x64-windows-dbg/drogon_ctl/test_cmake.h C:/vcpkg/buildtrees/drogon/x64-windows-dbg/drogon_ctl/test_cmake.cc 
cmd.exe /C "cd /D C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl && C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl\_drogon_ctl.exe create view C:/vcpkg/buildtrees/drogon/src/v1.8.4-f70ee9212f.clean/drogon_ctl/templates/test_cmake.csp"
[93/129] cmd.exe /C "cd /D C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl && C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl\_drogon_ctl.exe create view C:/vcpkg/buildtrees/drogon/src/v1.8.4-f70ee9212f.clean/drogon_ctl/templates/restful_controller_base_h.csp"
FAILED: drogon_ctl/restful_controller_base_h.h drogon_ctl/restful_controller_base_h.cc C:/vcpkg/buildtrees/drogon/x64-windows-dbg/drogon_ctl/restful_controller_base_h.h C:/vcpkg/buildtrees/drogon/x64-windows-dbg/drogon_ctl/restful_controller_base_h.cc 
cmd.exe /C "cd /D C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl && C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl\_drogon_ctl.exe create view C:/vcpkg/buildtrees/drogon/src/v1.8.4-f70ee9212f.clean/drogon_ctl/templates/restful_controller_base_h.csp"
[94/129] cmd.exe /C "cd /D C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl && C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl\_drogon_ctl.exe create view C:/vcpkg/buildtrees/drogon/src/v1.8.4-f70ee9212f.clean/drogon_ctl/templates/cmake.csp"
FAILED: drogon_ctl/cmake.h drogon_ctl/cmake.cc C:/vcpkg/buildtrees/drogon/x64-windows-dbg/drogon_ctl/cmake.h C:/vcpkg/buildtrees/drogon/x64-windows-dbg/drogon_ctl/cmake.cc 
cmd.exe /C "cd /D C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl && C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl\_drogon_ctl.exe create view C:/vcpkg/buildtrees/drogon/src/v1.8.4-f70ee9212f.clean/drogon_ctl/templates/cmake.csp"
[95/129] cmd.exe /C "cd /D C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl && C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl\_drogon_ctl.exe create view C:/vcpkg/buildtrees/drogon/src/v1.8.4-f70ee9212f.clean/drogon_ctl/templates/filter_h.csp"
FAILED: drogon_ctl/filter_h.h drogon_ctl/filter_h.cc C:/vcpkg/buildtrees/drogon/x64-windows-dbg/drogon_ctl/filter_h.h C:/vcpkg/buildtrees/drogon/x64-windows-dbg/drogon_ctl/filter_h.cc 
cmd.exe /C "cd /D C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl && C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl\_drogon_ctl.exe create view C:/vcpkg/buildtrees/drogon/src/v1.8.4-f70ee9212f.clean/drogon_ctl/templates/filter_h.csp"
[96/129] cmd.exe /C "cd /D C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl && C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl\_drogon_ctl.exe create view C:/vcpkg/buildtrees/drogon/src/v1.8.4-f70ee9212f.clean/drogon_ctl/templates/restful_controller_custom_cc.csp"
FAILED: drogon_ctl/restful_controller_custom_cc.h drogon_ctl/restful_controller_custom_cc.cc C:/vcpkg/buildtrees/drogon/x64-windows-dbg/drogon_ctl/restful_controller_custom_cc.h C:/vcpkg/buildtrees/drogon/x64-windows-dbg/drogon_ctl/restful_controller_custom_cc.cc 
cmd.exe /C "cd /D C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl && C:\vcpkg\buildtrees\drogon\x64-windows-dbg\drogon_ctl\_drogon_ctl.exe create view C:/vcpkg/buildtrees/drogon/src/v1.8.4-f70ee9212f.clean/drogon_ctl/templates/restful_controller_custom_cc.csp"
ninja: build stopped: subcommand failed.
```
</details>

**Additional context**

<details><summary>vcpkg.json</summary>

```
{
  "name": "sample-project",
  "version-string": "no-version",
  "dependencies": [
    {
      "name": "drogon",
      "features": [
        "ctl",
        "orm",
        "redis"
      ]
    }
  ]
}

```
</details>

--- a/ports/drogon/portfile.cmake
+++ b/ports/drogon/portfile.cmake
@@ -6,6 +6,7 @@ vcpkg_from_github(
     HEAD_REF master
     PATCHES
         vcpkg.patch
+        vcpkg_redis.patch
         drogon_config.patch
         fix_gcc13.patch #https://github.com/drogonframework/drogon/pull/1563
 )

--- a/ports/drogon/vcpkg.json
+++ b/ports/drogon/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "drogon",
   "version": "1.8.4",
-  "port-version": 1,
+  "port-version": 2,
   "description": "A C++14/17 based HTTP web application framework running on Linux/macOS/Unix/Windows",
   "homepage": "https://github.com/an-tao/drogon",
   "documentation": "https://drogon.docsforge.com/master/overview/",

--- a/ports/drogon/vcpkg_redis.patch
+++ b/ports/drogon/vcpkg_redis.patch
@@ -1,0 +1,69 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 4406362..62d7f50 100755
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -461,33 +461,28 @@ if (BUILD_SQLITE)
+ endif (BUILD_SQLITE)
+ 
+ if (BUILD_REDIS)
+-    find_package(Hiredis)
+-    if (Hiredis_FOUND)
+-        add_definitions(-DUSE_REDIS)
+-        target_link_libraries(${PROJECT_NAME} PRIVATE Hiredis_lib)
+-        set(DROGON_SOURCES
+-            ${DROGON_SOURCES}
+-            nosql_lib/redis/src/RedisClientImpl.cc
+-            nosql_lib/redis/src/RedisClientLockFree.cc
+-            nosql_lib/redis/src/RedisClientManager.cc
+-            nosql_lib/redis/src/RedisConnection.cc
+-            nosql_lib/redis/src/RedisResult.cc
+-            nosql_lib/redis/src/RedisTransactionImpl.cc
+-            nosql_lib/redis/src/SubscribeContext.cc
+-            nosql_lib/redis/src/RedisSubscriberImpl.cc)
+-        set(private_headers
+-            ${private_headers}
+-            nosql_lib/redis/src/RedisClientImpl.h
+-            nosql_lib/redis/src/RedisClientLockFree.h
+-            nosql_lib/redis/src/RedisConnection.h
+-            nosql_lib/redis/src/RedisTransactionImpl.h
+-            nosql_lib/redis/src/SubscribeContext.h
+-            nosql_lib/redis/src/RedisSubscriberImpl.h)
+-
+-    endif (Hiredis_FOUND)
+-endif (BUILD_REDIS)
+-
+-if (NOT Hiredis_FOUND)
++    find_package(hiredis CONFIG REQUIRED)
++    add_definitions(-DUSE_REDIS)
++    target_link_libraries(${PROJECT_NAME} PRIVATE hiredis::hiredis)
++    set(DROGON_SOURCES
++        ${DROGON_SOURCES}
++        nosql_lib/redis/src/RedisClientImpl.cc
++        nosql_lib/redis/src/RedisClientLockFree.cc
++        nosql_lib/redis/src/RedisClientManager.cc
++        nosql_lib/redis/src/RedisConnection.cc
++        nosql_lib/redis/src/RedisResult.cc
++        nosql_lib/redis/src/RedisTransactionImpl.cc
++        nosql_lib/redis/src/SubscribeContext.cc
++        nosql_lib/redis/src/RedisSubscriberImpl.cc)
++    set(private_headers
++        ${private_headers}
++        nosql_lib/redis/src/RedisClientImpl.h
++        nosql_lib/redis/src/RedisClientLockFree.h
++        nosql_lib/redis/src/RedisConnection.h
++        nosql_lib/redis/src/RedisTransactionImpl.h
++        nosql_lib/redis/src/SubscribeContext.h
++        nosql_lib/redis/src/RedisSubscriberImpl.h)
++else()
+     set(DROGON_SOURCES
+         ${DROGON_SOURCES}
+         lib/src/RedisClientSkipped.cc
+@@ -496,7 +491,7 @@ if (NOT Hiredis_FOUND)
+     set(private_headers
+         ${private_headers}
+         lib/src/RedisClientManager.h)
+-endif (NOT Hiredis_FOUND)
++endif (BUILD_REIDS)
+ 
+ if (BUILD_TESTING)
+     add_subdirectory(nosql_lib/redis/tests)

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -2194,7 +2194,7 @@
     },
     "drogon": {
       "baseline": "1.8.4",
-      "port-version": 1
+      "port-version": 2
     },
     "dstorage": {
       "baseline": "1.2.0",

--- a/versions/d-/drogon.json
+++ b/versions/d-/drogon.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "ddae8d79f08366e60ecaddf472fcf24a2c8c2ffd",
+      "version": "1.8.4",
+      "port-version": 2
+    },
+    {
       "git-tree": "29376f140626e1d34d6ca573b5a00e6dd7006d2f",
       "version": "1.8.4",
       "port-version": 1

